docs/IE_DG/ONNX_Support.md

```diff
@@ -9,10 +9,41 @@ InferenceEngine::Core core;
 auto network = core.ReadNetwork("model.onnx");
 ```
 
+**Reshape feature:**
+
 OpenVINO™ doesn't provide a mechanism to specify pre-processing (like mean values subtraction, reverse input channels) for the ONNX format.
 If an ONNX model contains dynamic shapes for input, please use the `CNNNetwork::reshape` method for shape specialization.
 
-Unsupported types of tensors:
+**Weights saved in external files:**
+
+OpenVINO™ supports ONNX models which use weights saved in external files. It is especially useful for models larger than 2GB because of protobuf limitations.
+If you want to read such model you should use `ReadNetwork` overload which takes `modelPath` as input parameter (both `std::string` and `std::wstring`).
+Note that `binPath` argument of `ReadNetwork` should be empty in such case, because paths to external weights are saved directly in an ONNX model.
+Otherwise the runtime exception will be thrown.
+Reading models with external weights is not supported by `ReadNetwork(const std::string& model, const Blob::CPtr& weights)` overload.
+
+The paths to external weights files are saved in a model file as relative to the model location.
+It means that if there is a model:
+`home/user/workspace/models/model.onnx`
+and a file which contains external weights:
+`home/user/workspace/weights.data`
```

Comment:
I don't think weights can be stored in a higher-level directory.
```suggestion
`home/user/workspace/models/data/weights.bin`
```

Reply:
It is supported (I am not sure if it is real scenario): https://github.com/openvinotoolkit/openvino/blob/58fa4b6f76bf7b9f64b2cf61fed9ffc29995b600/ngraph/test/models/onnx/external_data.prototxt#L44
For me it is good example because it emphasizes that the path is saved as relative

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/3098